pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -1,11 +1,31 @@
 # -*- coding: utf-8 -*-
 
+from itertools import chain
+from pkg_resources import resource_string
+import re
+
 import numpy as np
 
 from pyfr.integrators.dual.pseudo.base import BaseDualPseudoIntegrator
 from pyfr.util import proxylist
 
 
+def _get_coefficients_from_txt(scheme, nstages):
+    scheme = re.sub(r'(?<=\))\s*,?\s*(?!$)', r'\n', scheme)
+    a_mat = []
+
+    for i, l in zip(list(range(1, nstages + 1)), scheme.splitlines()):
```

Comment:
enumerate with start arg.

Reply:
Can also create `a` the old way and then do an `if any(len(ai) != i for i, ai in enumerate(a, start=1))` or something to that effect.